### PR TITLE
Docs/script subscribe to solution group

### PR DESCRIPTION
**Pull request by JGiter**
Example scripts to signup registrar, operator, register solution group and subscribe to it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ Use the follow script as an example to register solution.
 `ALICE` is one of the predefined development accounts. This script first
 signups ALICE as solution registrar, which allows next to register solution.
 
-```TS
```
Comment (JGiter) [inline README.md:192]: `TS` does not enables syntax highlighting

Comment (jrhender) [reply README.md:192]: Maybe give `ts` a try? I think typescript is in the list of supported languages by github:
- https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
- https://github.com/github-linguist/linguist/blob/7ca3799b8b5f1acde1dd7a8dfb7ae849d3dfb4cd/lib/linguist/languages.yml#L7150

**Review by jrhender — APPROVED** [review]
Small comment about syntax highlighting, but I think this looks good otherwise

File: README.md (hunk 1 above)
Comment (JGiter) [reply README.md:192]: yes, `ts` is working

**Pull request merged**